Doc/library/ssl.rst

```diff
@@ -764,6 +769,15 @@ Constants
 
    .. versionadded:: 3.4
 
+.. data:: OP_NO_TLSv1_3
+
+   Prevents a TLSv1.3 connection. This option is only applicable in conjunction
+   with :const:`PROTOCOL_TLS`. It prevents the peers from choosing TLSv1.3 as
+   the protocol version. TLS 1.3 is available with OpenSSL 1.1.1 or later.
+   With older versions, the flag defaults to *0*.
```

Comment:
The reasonable default for ```create_default_context()``` is TLS 1.3 enabled. I've been doing TLS 1.3 tests with NSS and OpenSSL for various products as part of my day job. It works mostly fine but not every time. People will have to disable TLS 1.3 explicitly. For example see @njsmith issue https://github.com/pyca/pyopenssl/issues/624